guide/payments/request.md

```diff
@@ -116,82 +88,120 @@ A piece of meta data unique to invoices is an expiration time. This is how long
    image = "/assets/images/guide/payments/request/single-use-payment-request.png"
    retina = "/assets/images/guide/payments/request/single-use-payment-request@2x.png"
    layout = "shadow"
-   caption = "A newly generated single-use Lightning payment request."
+   caption = "A newly generated single-use payment request containing both an invoice and a on-chain address."
    alt-text = "Screen showing single-use payment request."
    width = 250
    height = 541
 %}
 
+</div>
+
+### Amounts
+
+When generating single-use payment requests, the receiver may want to add an amount to be paid. Entering an amount from the home screen is recommended.
+
+Zero amount single-use payment requests should also be easily generated from the home screen by simply tapping request with no amount entered. By allowing the sender to set the amount when they are ready to pay reduces both parties exposure to price volatility, as the amount paid in bitcoin will be more accurate in dollars.
+
+Units must be toggleable between bitcoin, satoshi and the users local currency. Ensure the unit being entered is clear to prevent entering of the wrong unit amount. More on the [Units & Symbols page]({{ "/guide/payments/units-and-symbols/" | relative_url }}).
```

Comment:
```suggestion
Units must be toggleable between bitcoin, satoshi and the users local currency. Ensure the unit being entered is clearly visible, to prevent entering the wrong unit amount. More on the [Units & Symbols page]({{ "/guide/payments/units-and-symbols/" | relative_url }}).
```